packages/server/lib/browsers/cdp_automation.ts

```diff
@@ -188,6 +189,15 @@ export class CdpAutomation {
     onFn('Network.responseReceived', this.onResponseReceived)
   }
 
+  async startVideoRecording (writeVideoFrame: WriteVideoFrame, screencastOpts?) {
+    this.onFn('Page.screencastFrame', async (e) => {
+      writeVideoFrame(Buffer.from(e.data, 'base64'))
+      await this.sendDebuggerCommandFn('Page.screencastFrameAck', { sessionId: e.sessionId })
```

Comment:
I see what you mean, but I prefer it this way, since I don't want to "return" any value. A synchronous function wouldn't `return` on this line either. Absent of a coding style guide, there's a mix of both ways in the codebase.